### ISSUE TITLE
Bug in user-bootstrap.sh: Incorrect Git Clone URL for Mininet

### ISSUE THREAD

**Issue by Anshul-Johri-1** [p4lang/tutorials#553]
While working on setting up the P4 tutorials environment, I encountered a bug in the [user-bootstrap.sh at Line 17](https://github.com/p4lang/tutorials/blob/master/vm/user-bootstrap.sh#L17). 
The script attempts to clone the Mininet repository using the Git protocol, which results in a connection timeout error. Though it's only for historical reference, but it still needs to be corrected.

**Bug:** 
```
git clone git://github.com/mininet/mininet mininet
```

**Corrected Command:**
```
git clone https://github.com/mininet/mininet mininet
````

**Error Message Received**
```
fatal: unable to connect to github.com:
github.com[0: 20.207.73.82]: errno=Connection timed out
```

**Comment by AdarshRawat1** [comment]
@Anshul-Johri-1 Thank you for pointing that out.

**Comment by jafingerhut** [comment]
Fixed by PR #554
